skbio/io/format/tests/test_stockholm.py

```diff
@@ -853,8 +853,8 @@ def test_gr_wrong_dataframe_item_length_error(self):
                               positional_metadata=pos_metadata_dataframe)])
         with self.assertRaisesRegex(StockholmFormatError,
                                     r'Sequence-specific positional metadata.*'
-                                    'must contain a single character.*Found '
-                                    'value\(s\) in column AC'):
+                                    r'must contain a single character.*Found '
```

Comment:
It seems like the error messages are missing a space. For example:
- here: `character.*Found`
- the test below `character.*Found value`
is that right? If it is, can we fix?

Reply:
@antgonza note that this is a regex check -- that * is ignoring characters between the `.` and `Found`

Reply:
duh! thanks!